frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import PropTypes from "prop-types";
+import { Card, Row, Col, Select } from "antd";
+import RenderSelect from "@/components/common/RenderSelect";
+
+import ConditionalButton from "@/components/common/ConditionalButton";
+import { modalConfig } from "@/components/modalContent/config";
+import * as String from "@/constants/strings";
+import * as ModalContent from "@/constants/modalContent";
+
+import { fetchPartyRelationshipTypes } from "@/actionCreators/partiesActionCreator";
+import { getPartyRelationshipTypes } from "@/selectors/partiesSelectors";
+
+const propTypes = {
+  openModal: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  handlePartySubmit: PropTypes.func.isRequired,
+  fetchPartyRelationshipTypes: PropTypes.func.isRequired,
+  partyRelationshipTypes: PropTypes.array.isRequired,
+};
+
+const defaultProps = {
+  expectedDocumentStatusOptions: [],
+  partyTypeOptions: [
+    {
+      value: "EoR",
+      label: "Engineer on Record",
+    },
+    {
+      value: "MM",
+      label: "Mine Manager",
+    },
+  ],
+};
+
+export class ViewPermittee extends Component {
+  componentDidMount() {
+    this.props.fetchPartyRelationshipTypes();
+  }
+
+  onSubmitAddPartyRelationship = (values) => {
+    /*     this.props
```

Comment:
Please remove all the commented-out code in this file.

Reply:
This is WIP code that we want to carry forward.

Reply:
Un-commented.